membership-common/src/test/scala/com/gu/memsub/subsv2/reads/CatReadsTest.scala

```diff
@@ -70,16 +70,12 @@ class CatReadsTest extends AnyFlatSpec {
               Map(
                 GBP -> Price(74.4f, GBP),
                 USD -> Price(90f, USD),
-                CAD -> Price(86.25f, CAD),
-                AUD -> Price(106.0f, AUD),
-                NZD -> Price(132.5f, NZD),
-                EUR -> Price(67.5f, EUR),
```

Comment:
to be honest this was a strange one.  I can understand someone changed the DEV zuora catalog to make it clear the 3 month fixed term GW is a gift, but why did some of the prices go away?  In actual zuora dev I can see it's only got GBP and USD via the API
```
                            "pricingSummary": [
                                "USD90",
                                "GBP74.4"
                            ],
```
However if I look in the UI I can see the other prices are there, just not active for some reason (the activate button is there)
![image](https://github.com/guardian/members-data-api/assets/7304387/a3beeb7f-bd90-48cc-b41d-ccf9444b8fe7)


This test was failing before my changes, but because the membership-common tests weren't automated, it hadn't been noticed.